src/components/editTranscript/EditTranscript.tsx

```diff
@@ -84,6 +83,7 @@ const EditTranscript = ({
   // restoreOriginal content function
   const onClickRestore = () => {
     update(data.originalContent?.body || "");
+    editorRef.current?.setText(data.originalContent?.body);
```

Comment:
If this is the fix, have a look at the restoreOriginal function to see what is broken and perhaps put the fix there

Reply:
We don't want to have redundant functions

Reply:
this would mean i would have to pass the useRef from the parent component.